proposals/stage-0/MEMBERSHIP_PROGRAM/README.md

```diff
@@ -0,0 +1,62 @@
+# Individual Membership Representation Proposal
+>  Stage 0
+
+## Champion
+
+[Sara Chipps](https://github.com/sarajo)
+ 
+## Description
+
+The goals of the Individual Membership Program are to help bring awareness about both large and small projects to possible contributors. A way for individuals to show support and be a part of the OpenJS foundation, as well as providing an opportunity for community with the other members. 
+
+While one goal of this group is introducing members to projects and helping the projects find the support they need; the membership program will also be an opportunity to welcome an inclusive group of coders from around the world that love the projects that are part of the OpenJS foundation. This also brings in new perspectives and feedback from the vast community of JavaScript developers.
+
+Additionally, Individual membership should help people understand that their projects too can get support from the foundation, and the OpenJS foundation is their foundation. 
+A tertiary goal is to establish a pool from which an ambassador program can be started. Recognizing people that evangelize and recruit for OpenJS projects. 
+
+## Membership Benefits
+
+Members of this program [name TBD] will receive an members-only weekly newsletter keeping them up to date on the last from OpenJS projects, the CPC, and the board. They will be invited to participate in discussions about governance and new initiatives. 
+
+They will be featured on our global membership page, and will be given a digital badge that they can put on an avatar, their blog, and/or personal website. 
+
+Additionally, they will receive discounts for training, certification, conferences, and potentially exclusive offers from member organizations. 
+
+Lastly, members will have access to the member store where they can purchase OpenJS member swag to show their support for our projects. Revenue from swag will go towards furthering the reach of the foundation. 
```

Comment:
```suggestion
Lastly, members will have access to the member store where they can purchase OpenJS Foundation member swag to show their support for our projects. Revenue from swag will go towards furthering the reach of the foundation. 
```

Reply:
can we also change all the `OpenJS` reference to `OpenJS Foundation`? I think somewhere we decided to stick to that naming.